src/runtime_src/core/pcie/driver/linux/xocl/subdev/mb_scheduler.c

```diff
@@ -2531,8 +2531,21 @@ exec_mark_cmd_state(struct exec_core *exec, struct xocl_cmd *xcmd, enum ert_cmd_
 static inline void
 exec_mark_cmd_complete(struct exec_core *exec, struct xocl_cmd *xcmd)
 {
+	//For soft kernels get cmd state and return_code
```

Comment:
prefer /* */ than // in kernel code
https://kernel.googlesource.com/pub/scm/linux/kernel/git/apw/checkpatch/+/v3.2-rc1/Documentation/CodingStyle#443

Reply:
done